concepts/05 Widgets/Scheduler/020 Data Binding/10 Simple Array/15 ArrayStore.md

```diff
@@ -42,11 +42,74 @@ If you want to extend the functionality of a JavaScript array, place it into the
         [dataSource]="appointmentStore">
     </dx-scheduler>
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+          :data-source="dataSource"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler from 'devextreme-vue/scheduler';
+
+    const appointments = [ /* ... */ ];
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                dataSource: new DevExpress.data.ArrayStore({
```

Comment:
Sadly, it doesn't work like that. You need to import `ArrayStore` and then create it using `new ArrayStore`.